terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/validation/ValidationTest.java

```diff
@@ -0,0 +1,265 @@
+/*

```

Comment:
Sorry, It's difference that may occur due to I had to rebase too much.
I Canceled the wrong rebase, and re-push only the necessary commits.
This difference does not occur.